docs/topics/install-jboss-tools.adoc

```diff
@@ -0,0 +1,17 @@
+[[install_jboss_tools]]
+= Install JBoss Tools into Eclipse
```

Comment:
It is not a requirement to install JBoss Tools, only that Eclipse knows where to find those JBoss Tools dependencies when it's installing our plugin. 

So there are three choices 

1) install using DevStudio, which already has JBoss Tools installed by default, thus Eclipse knows where to find the deps or
2) install using a vanilla distro of Eclipse and our composite update site (http://download.jboss.org/jbosstools/photon/stable/updates/rhamt/), which will automatically detect and pickup JBoss Tools or
3) install using a vanilla distro of Eclipse and our direct update site (http://download.jboss.org/jbosstools/photon/stable/updates/rhamt/4.2.0.Final/) and then manually add the JBoss Tools update site location (eg https://devstudio.redhat.com/10.0/stable/updates/)

Reply:
I'm attempting #2 above, using the following steps:

1. Install a fresh copy of Eclipse Photon.
2. Go to Help -> Install New Software
3. Click Add
4. In `Name`, define `RHAMT`.
5. In `Location`, define `http://download.jboss.org/jbosstools/photon/stable/updates/rhamt/`.
6. Click Add
7. Check `JBoss Tools - RHAMT`
8. Click Next

At this point I get the same error reported in WINDUP-2245 regarding missing dependencies. Installing JBoss Tools manually, or using JBDS, resolves this issue.

Is there a step I'm missing?